actions/utils/dg-deploy-init/action.yaml

```diff
@@ -0,0 +1,48 @@
+name: "Initialize Deploy Session"
+description: "Initialize a new deploy session to Dagster+"
+inputs:
+  project_dir:
+    required: true
+    description: "The root directory of the dg project or workspace."
+  deployment:
+    required: false
+    description: "The deployment name to deploy. For pull requests, this deployment will be used as the base deployment."
+    default: "prod"
+  location_names:
+    required: false
+    description: "JSON list containing names of locations to deploy. If unspecified, all locations are deployed."
+
+runs:
+  using: "composite"
+  steps:
+    - name: init-env-vars
+      run: >
+        echo "DAGSTER_CLOUD_PEX=$GITHUB_ACTION_PATH/../../../generated/gha/dagster-cloud.pex" >> $GITHUB_ENV &&
+        echo "DAGSTER_BUILD_STATEDIR=/tmp/statedir-$GITHUB_RUN_ID" >> $GITHUB_ENV
```

Comment:
do you still need to specify the statedir?

Reply:
you don't need to, but i don't think it hurts?

Reply:
Put it as a nit, it does not hurt now, maybe later. 

Your CLI was setting a default. Now, every step used with the CLI will need to set the same statedir?
If we have a pattern of utils/dg-cli to allow for adhoc command, the users will need to set that same statedir for the commands that need it?

Reply:
but I suppose you're doing this to be compatible with Dagster Cloud for the notify/status action, might be something just to remove later